### PR TITLE
Added spotify to the available login/signup modules

### PR DESCRIPTION
**Pull request by 3nvi**

### CODE REVIEW

**Comment by 3nvi** [comment]
Only added the basic GET endpoints that match the purpose of the hello.js library

I currently don't have a Browserstack key to test it using this service, but i would gladly do if you could help me set it up

**Comment by MrSwitch** [comment]
Fantastic, can we get this without editing the dist/ folder?

**Comment by 3nvi** [comment]
Sure, i' ll let you know

**Comment by 3nvi** [comment]
Hey, i have:

* tested it locally and on browserstack
* did some small updates to handle the edge cases
* reverted any commits that changed the `/dist` content. 

If you want a --hard reset on the `/dist` commits, i can open a new PR from a different branch that cherry-picks only the spotify additions.

Let me know
Thanks :)

File: demos/client_ids.js [hunk 1]
```diff
@@ -48,7 +48,10 @@ var TWITTER_CLIENT_ID = {
 var SOUNDCLOUD_CLIENT_ID = {
 	'local.knarly.com' : '8a4a19f86cdab097fa71a15ab26a01d6',
 	'adodson.com' : location_https ? 'eacfb4d662e046c19e53c9e44bf7f7b7' : '47a386647dadf913e559c12ef6db4292'
-}[window.location.hostname];
+}[window.location.hostname];0561613da0214b339781fd2c5c005382
+
+// Spotify Register - https://developer.spotify.com/my-applications/#!/applications/create
+var SOUNDCLOUD_CLIENT_ID = '0561613da0214b339781fd2c5c005382';
```
Comment (MrSwitch) [inline demos/client_ids.js]: Probably remove this, i'll need to register some to make it work in the demo

**Comment by 3nvi** [comment]
Reverted all changes to `client_ids` file. You do need to create a spotify app for the demos :)

**Pull request merged**